### PR TITLE
[xorm] Remove cache from xorm

### PR DESCRIPTION
**Pull request by ying-jeanne**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/util/xorm/session_get.go [hunk 1]
```diff
@@ -124,7 +94,7 @@ func (session *Session) nocacheGet(beanKind reflect.Kind, table *core.Table, bea
 			return true, err
 		}
 		if res.Valid {
-			*(bean.(*string)) = res.String
+			*(bean) = res.String
```
Comment (sakjur) [inline pkg/util/xorm/session_get.go:97]: I don't think you need the parentheses here, fwiw

File: pkg/util/xorm/session_convert.go [hunk 2]
```diff
@@ -17,6 +17,11 @@ import (
 	"xorm.io/core"
 )
 
+const (
+	zeroTime0 = "0000-00-00 00:00:00"
+	zeroTime1 = "0001-01-01 00:00:00"
```
Comment (sakjur) [inline pkg/util/xorm/session_convert.go:22]: Are these used?

File: pkg/util/xorm/session_insert.go [hunk 3]
```diff
@@ -518,7 +511,7 @@ func (session *Session) innerInsert(bean interface{}) (int64, error) {
 		}
 		defer handleAfterInsertProcessorFunc(bean)
 
-		session.cacheInsert(tableName)
+		// session.cacheInsert(tableName)
```
Comment (sakjur) [inline pkg/util/xorm/session_insert.go:515]:
```suggestion
```

**Review by sakjur — APPROVED** [review]

File: pkg/util/xorm/session_convert.go (hunk 2 above)
Comment (ying-jeanne) [reply pkg/util/xorm/session_convert.go:22]: yes in this file, i just deleted the time_helper, since all other things inside is not used.

**Review by zserge — APPROVED** [review]

**Pull request merged**